napari_segment_blobs_and_things_with_membranes/__init__.py

```diff
@@ -944,3 +987,139 @@ def sub_sample(image:"napari.types.ImageData", sample_x: int = 1, sample_y: int
 @time_slicer
 def squeeze(image:"napari.types.ImageData") -> "napari.types.ImageData":
     return np.squeeze(image)
+
+
+@register_function(menu="Segmentation post-processing > Grayscale erosion (scikit-image, nsbatwm)")
```

Comment:
I presume Greyscale erosion is a filter (the name sounds like). Thus, it should be in a different menu. Furthermore, I'm wondering, is the underlying math identical with the `minimum_filter` ?

Reply:
Good point! I could move all the grayscale operations into `Filters/background removal`, and keep the binary ones under `Segmentation post-processing`? The only issue is that the grayscale operations can be quite useful for both `napari.types.ImageData` and  `napari.types.LabelData` e.g. grayscale erosion can be useful to erode a label image while retaining all the individual labels. Is there any way to allow a function to take both `napari.types.ImageData` and `napari.types.LabelsData` and potentially appear in two menus? It's not a big problem if not - it's easy to re-run connected components to get individual labels again.

Yes - I think grayscale erosion is equivalent to `minimum_filter`, as grayscale dilation is equivalent to `maximum_filter`. So these grayscale operations could be removed, as long as they were changed to use a disk footprint.